mattermost_connector.py

```diff
@@ -1659,10 +1705,12 @@ def finalize(self):
 
             headers = dict()
             headers['Cookie'] = 'csrftoken={}'.format(csrftoken)
-            headers['Referer'] = 'https://127.0.0.1/login'
+            headers['Referer'] = BaseConnector._get_phantom_base_url() + \
+                "login"
```

Comment:
Is this really exceeded the character limit for the line? It looks like it should fit. 😅

Reply:
Done.